package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "redash-client",
-  "version": "10.1.2",
+  "version": "10.1.4",
```

Comment:
10.1.3 was used for different purpose and tag already present in dockerhub. So changing to 10.1.4